api/src/fs.ts

```diff
@@ -1,4 +1,4 @@
-import { promisified } from './tauri'
+import { promisified } from "./tauri";
```

Comment:
You are changing our style. Please revert `"` to `'` and `no-semis` as well as `no-trailing-commas`

Reply:
That is why eslint is failing.
https://github.com/tauri-apps/tauri/pull/850/checks?check_run_id=1860478292

Reply:
i've seen this change happening by accident several times, maybe it's something with vscode/prettier

Reply:
ah, it was a missing prettier config on the api module :)